app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Name:</strong>
+  <%= @recipe.name %>
+</p>
+
+<p>
+  <strong>Preparation time:</strong>
+  <%= @recipe.preparation_time %>
+</p>
+
+<p>
+  <strong>Cooking time:</strong>
+  <%= @recipe.cooking_time %>
+</p>
+
+<p>
+  <strong>Description:</strong>
+  <%= @recipe.description %>
+</p>
+
+<p>
+  <strong>Public:</strong>
+  <%= @recipe.public %>
+</p>
+
+<p>
+  <strong>User:</strong>
+  <%= @recipe.user_id %>
+</p>
+
+<%= link_to 'Edit', edit_recipe_path(@recipe) %> |
+<%= link_to 'Back', recipes_path %>
+<%= link_to 'Add ingredient', new_recipe_recipe_food_path(@recipe) %>
```

Comment:
- [ ] To be implemented.

- On the recipe show page, you should be able to generate a shopping list.

Optional suggestion

- [ ] Instead of clicking show to display the `Add ingredient` option, please make one of the items to be a link when a user clicks on the click can see that option.